docs/UserGuide.md

```diff
@@ -175,18 +244,25 @@ _Details coming soon ..._
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains 
+the data of your previous AddressBook home folder.
```

Comment:
i think AddressBook should be DevEnable?

Reply:
Changed. Thank you!